source/connection.cc

```diff
@@ -118,6 +118,19 @@ RestClient::Connection::SetUserAgent(const std::string& userAgent) {
 }
 
 /**
+ * @brief set custom Certificate Authority (CA) path
+ * default restclient-cpp/RESTCLIENT_VERSION string
+ *
+ * @param caInfo - The path to a file holding the certificates used to
+ * verify the peer with. See CURLOPT_CAINFO
+ *
+ */
+void
+RestClient::Connection::SetCAInfo(const std::string& caInfo) {
+  this->caInfo = caInfo;
+}
```

Comment:
I think something like `SetCAInfoFilePath` for the method and the corresponding instance variable make it more clear what the argument should actually be (filepath vs actual contents as string)